src/main/kotlin/hu/kirdev/schpincer/web/ApiController.kt

```diff
@@ -336,6 +336,22 @@ open class ApiController(
 
     private val trashpandaVoters: ConcurrentHashMap<String, Int> = ConcurrentHashMap<String, Int>()
 
+    @PostMapping("/provider/{circle}/increaseInterest")
+    @ResponseBody
+    fun circleInterestPlusPlus(@PathVariable circle: String, model: Model, request: HttpServletRequest): String {
```

Comment:
This should be in a transaction

Reply:
I'll pull the transactional part to service layer, as it is recommended (as I see it, and as it was pointed out here: https://stackoverflow.com/a/18637119 ).

Reply:
(Done)